### ISSUE TITLE
Highlight invalid comma after last array element

### ISSUE THREAD

**Issue by dominikbraun** [cflint/CFLint#620]
This feature proposal refers to [the issue i submitted](https://github.com/KamasamaK/vscode-cfml/issues/34) to vscode-cfml. Since the issue my proposal addresses is engine-specific, it is rather an improvement for CFLint than for the CFML plugin.

In CFScript, the last element of an array isn't allowed to have a trailing comma. Trailing commas after the last element are considered as a syntax error in Adobe ColdFusion.

`coffee = [ "Cappuccino", "Espresso", "Latte macchiato", ];`

This code will result in an error (except for Lucee/Railo). Would it be great effort to highlight/underline the closing bracket in order to indicate that the trailing comma is not valid?

**Comment by KamasamaK** [comment]
I propose that the rule(s) be configurable with values `always` and `never`.

**Comment by ryaneberly** [comment]
Is having a trailing comma a style that some prefer?

**Comment by KamasamaK** [comment]
@ryaneberly Yes, it is. Here's an example of the TSLint Rule: [trailing-comma](https://palantir.github.io/tslint/rules/trailing-comma/).

Based on #263 and #365, it seems likely that @mpaluchowski (or someone on their team) is one of those people who prefer it.

**Comment by dominikbraun** [comment]
@ryaneberly Having a trailing comma is preferred by some (or many?) Lucee users since it makes adding and shifting lines easier. However, it's simply invalid in Adobe ColdFusion and this feature was requested by the Lucee community.

**Comment by dominikbraun** [comment]
What's the current state of this issue? :smile:
